### DISCUSSION TITLE
standalone目录下仓库是同步更新的吗

### DISCUSSION THREAD

**Discussion by yinghaoyu** [alibaba/yalantinglibs#678] [Q&A] [answered]
请问`yalantinglibs/include/ylt/standalone/iguana`与`https://github.com/qicosmos/iguana`是同步更新的么？看提交好像不一样啊

**Comment by qicosmos** [comment, marked as answer]
是定期同步的，ylt的代码可能会稍微落后一点。

**Reply by yinghaoyu** [reply]
感谢回答
